xml/System.Runtime.Versioning/RequiresPreviewFeaturesAttribute.xml

```diff
@@ -20,7 +20,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>An attribute to decorate features that are in preview mode.</summary>
```

Comment:
@pgovind this wasn't in triple slash, I added it so that all undoc APIs in this file would be documented. But I am not sure if this is the best description, or even a complete one.

The documentation seems quite extensive: https://github.com/dotnet/designs/blob/main/accepted/2021/preview-features/preview-features.md

Can you help me provide a more complete summary to describe this attribute?

Reply:
How about something like:
`This attribute declares an API to be Preview and allows call sites to be flagged with a diagnostic indicating Preview API usage. Authors can use this attribute to ship Preview features in their assemblies.`

Reply:
Yeah, much better @pgovind. Should there also be a link to the document from my previous comment?

```suggestion
    <summary>This attribute declares an API to be Preview and allows call sites to be flagged with a diagnostic indicating Preview API usage. Authors can use this attribute to ship Preview features in their assemblies.</summary>
```